profiler/Directory.Build.props

```diff
@@ -189,13 +189,18 @@
       <SHARED-LIB-PATH>$(DOTNET-TRACER-REPO-ROOT-PATH)shared\src\native-lib\</SHARED-LIB-PATH>
       <CORECLR-PATH>$(SHARED-LIB-PATH)coreclr\src</CORECLR-PATH>
       <SHARED-LIB-INCLUDES>$(SHARED-LIB-PATH)fmt\include;$(SHARED-LIB-PATH)spdlog\include</SHARED-LIB-INCLUDES>
+      <!-- VCPKG setup -->
+      <VcpkgEnableManifest>true</VcpkgEnableManifest>
+      <VcpkgUseStatic>true</VcpkgUseStatic>
+      <VcpkgIntermediateFolder>$(DOTNET-TRACER-REPO-ROOT-PATH)obj\vcpkg</VcpkgIntermediateFolder>
```

Comment:
As mentioned earlier, would love to move this into `artifacts/obj` or `artifacts/deps` or anything really 😄